src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -414,8 +418,35 @@ class ComposterOverlay {
             list.add(Renderable.link("$name§r §8x${itemsNeeded.addSeparators()} §7(§6$format§7)") {
                 onClick(internalName)
                 if (LorenzUtils.isControlKeyDown()) {
-                    inInventory = false
-                    BazaarApi.searchForBazaarItem(itemName, itemsNeeded.toInt())
+                    if(lastAttemptTime.passedSince()>0.5.seconds){
+                        lastAttemptTime = SimpleTimeMark.now()
+                        val amountInSacks = fetchSackItem(internalName.asInternalName())?.amount
+                        val sacksLoaded = fetchSackItem(internalName.asInternalName())?.outdatedStatus
```

Comment:
instead of calling fetchSackItem twice it would be better to call it once and store that as a seperate variable then set both of these varaibles later.

```                        
val itemData = fetchSackItem(internalName.asInternalName())
val amountInSacks = itemData?.amount
val sacksLoaded = itemData?.outdatedStatus
```